x-pack/test_utils/stub_web_worker.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+if (!window.Worker) {
+  // @ts-ignore we aren't honoring the real Worker spec here
+  window.Worker = function Worker() {
+    this.postMessage = jest.fn();
+
+    // @ts-ignore TypeScript doesn't think this exists on the Worker interface
+    // https://developer.mozilla.org/en-US/docs/Web/API/Worker/terminate
+    this.terminate = jest.fn();
+  };
+}
```

Comment:
Is this being used? I didn't see anything in your code to indicate that it's using a web worker.
I was wondering if this was perhaps a holdover from an earlier iteration that used a web worker?

Reply:
This is being imported into a couple of test suites which end up rendering the JSON rule editor. The underlying `EuiCodeEditor` component uses `brace`/`ace` under the hood, which in turn uses web workers:

Ex:

https://github.com/elastic/kibana/blob/17d77c9d396cd3f97e681d8b053f5f877114b0c3/x-pack/legacy/plugins/security/public/views/management/role_mappings/edit_role_mapping/components/edit_role_mapping_page.test.tsx#L11-L14

Reply:
Oh, okay thanks!